.circleci/config.yml

```diff
@@ -65,7 +65,7 @@ jobs:
     steps:
       - attach_workspace:
           at: '~/figgy'
-      - run: sudo apt-get install gdal-bin libgdal-dev libcairo2-dev libpango1.0-dev
+      - run: sudo apt-get install gdal-bin libgdal-dev libcairo2-dev libpango1.0-dev tesseract-ocr tesseract-ocr-ita tesseract-ocr-eng
```

Comment:
Add ticket for corresponding ansible updates?

Reply:
Should merging this wait until ansible has been updated / run?

Reply:
https://github.com/pulibrary/princeton_ansible/pull/249